src/redux/books/book.js

```diff
@@ -42,5 +23,61 @@ const booksReducer = (state = initState, action) => {
   }
 };
 
+const apiBooksFetched = (payload) => ({
+  type: API_BOOKS_FETCHED,
+  payload,
+});
+
+const addBookAction = (payload) => ({
+  type: BOOK_ADDED,
+  payload,
+});
```

Comment:
[REQUIRED] Your addBook is not functioning properly.  It only displays a button unless we refresh.

|Your Bookstore|
|-----------------|
|![bookstore1](https://user-images.githubusercontent.com/62938735/193419505-eaac0ece-280f-4e21-a862-bf8e174e06e2.gif)|